Source/Logger/trace.cpp

```diff
@@ -156,7 +147,7 @@ void TraceMessageToDebugger(
 
     char outputBuffer[BUFFER_SIZE] = {};
     // [threadId][level][time][area] message
-    auto written = stprintf_s(outputBuffer, "[%04X][%s][%02d:%02d:%02d.%03u][%s] %s",
+    auto written = stprintf_s(outputBuffer, "[%04" PRIX64 "][%s][%02d:%02d:%02d.%03u][%s] %s",
```

Comment:
have you tried `llX`?
I would like to avoid using the macro type specifiers if possible

Reply:
llX works, I can change it to use that.